src/db/migrations/v1.8.1-add-partnership-details.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("partners", {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      is_primary_contact: {
+        type: Sequelize.BOOLEAN
+      },
+      operatorId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: "operators",
+          key: "id"
+        },
+        onUpdate: "cascade",
+        onDelete: "cascade"
+      }
```

Comment:
All the other tables seem to have a created and updated date time field so I guess we should continue that - not sure how it is used but we can figure that out.